pkgs/npm/src/cli.ts

```diff
@@ -328,8 +335,13 @@ async function run(): Promise<void> {
       throw new Error("Could not find the defang executable.");
     }
 
+    const commandline = ["npx", getEndNameFromPath(pathToExec)]
```

Comment:
I couldn't find a way to tell the difference between a npm exec, npx, or global install of defang cli. Will use npx as part of the hint.